src/index.ts

```diff
@@ -102,6 +105,14 @@ export class Client {
     return accountIds[0];
   }
 
+  public getPrimaryAccountId(capability = 'urn:ietf:params:jmap:mail'): string {
+    const primaryAccounts = this.session?.primaryAccounts;
```

Comment:
```suggestion
    const primaryAccounts = this.getSession().primaryAccounts;
```
Here you can use the session getter to inform the user (throw an exception) if the session is not fetched yet.